packages/salesforcedx-vscode-core/package.json

```diff
@@ -749,6 +750,11 @@
           "type": "boolean",
           "default": false,
           "description": "%conflict_detection_enabled_description%"
+        },
+        "salesforcedx-vscode-core.beta-perf-enhancements": {
```

Comment:
<img width="388" alt="Screen Shot 2020-03-09 at 5 55 11 PM" src="https://user-images.githubusercontent.com/10779873/76271504-fb78ad00-6235-11ea-8947-53e4612636a1.png">

Totally a nit but I suggest updating the setting name to be more descriptive - It's a bit odd to read. Examples: `"salesforcedx-vscode-core.performanceEnhancements"` or `"salesforcedx-vscode-core.deployAndRetrieveEnhancements"` and put "beta" in the description. It might also be nice to write that enabling the setting may improve the experience by speeding up deploy times. As a customer I might not know what the implication is behind "using the tooling api for single file deploys"

Reply:
I agree, I'm going to change the description to be more generic so we can use it to gate all the new perf enhancements